src/course-team/utils.js

```diff
@@ -0,0 +1,44 @@
+import { MODAL_TYPES } from './enum';
+import messages from './info-modal/messages';
+
+const getErrorEmailFromMessage = (message) => {
+  const regex = /'([^']+)'/g;
+  return message.match(regex)[0];
+};
+
+const getInfoModalSettings = (modalType, currentEmail, errorEmail, courseName, intl) => {
```

Comment:
Since we do not use typescript in this project, I propose to write for the functions that we put in the utilities the description of JSDocs documentatin as on the Advanced settings page